configure.ac

```diff
@@ -41,6 +41,10 @@ AS_IF([test "x$enable_jni" = "xyes"], [
 ])
 AM_CONDITIONAL([JAVAI], [test "x$enable_jni" = "xyes"])
 
+# Enable curses library
+AX_WITH_CURSES
+AS_VAR_APPEND(LDFLAGS, ["$CURSES_LIB"])
```

Comment:
I think you're appending $CURSES_LIB to the wrong variable. You can make everything build, including the package, by appending to LIBS instead. I do not understand the intended uses for LIBS/LDFLAGS/xxx_LDADD/etc so there may be a better way.